commons-jcs-core/src/main/java/org/apache/commons/jcs3/engine/CacheWatchRepairable.java

```diff
@@ -58,22 +58,16 @@
     public void setCacheWatch( final ICacheObserver cacheWatch )
     {
         this.cacheWatch = cacheWatch;
-        cacheMap.entrySet().forEach(entry -> {
-            final String cacheName = entry.getKey();
-            entry.getValue().forEach(listener -> {
-                try
-                {
-                    log.info( "Adding listener to cache watch. ICacheListener = "
-                            + "{0} | ICacheObserver = {1}", listener, cacheWatch );
-                    cacheWatch.addCacheListener( cacheName, listener );
-                }
-                catch ( final IOException ex )
-                {
-                    log.error( "Problem adding listener. ICacheListener = {0} | "
-                            + "ICacheObserver = {1}", listener, cacheWatch, ex );
-                }
-            });
-        });
+        cacheMap.forEach((cacheName, value) -> value.forEach(listener -> {
```

Comment:
Please use the code formatting conventions of the project.

Reply:
HI @tvand 
I think it's solved.
TY